packages/manager/modules/octavia-load-balancer/src/onboarding/constants.js

```diff
@@ -1,3 +1,169 @@
-export const GUIDES = [];
+export const GUIDES = [
+  {
+    title: 'octavia_onboarding_guide1_title',
+    description: 'octavia_onboarding_guide1_description',
+    links: {
+      DEFAULT: 'https://docs.ovh.com/gb/en/storage/nas/get-started/',
```

Comment:
**question:** Why default link is different and pointing to NAS?

Reply:
I forgot to change it :)